packages/caver-core-method/src/index.js

```diff
@@ -762,28 +762,37 @@ const checkForContractDeployment = (mutableConfirmationPack, receipt, sub) => {
         return
     }
 
-    if (!receipt.status && receipt.txError) {
+    if (!Boolean(utils.hexToNumber(receipt.status)) && receipt.txError) {
```

Comment:
```suggestion
    if (!utils.hexToNumber(receipt.status) && receipt.txError) {
```
This might fix the lint error. My bad.